README.md

```diff
@@ -2,16 +2,21 @@
 API
 ======
 
+A nodejs service that does the following:
+- Authorizes and validates requests. 
+- Creates and starts gem2s and qc state machines for Data Processing module.
+- Creates a SQS queue and assigns available worker to an experiement that needs one for Data Exploration tasks.
```

Comment:
We need the worker for clustering (in data processing) and the dot plot (in plots and tables) too